blog/index.html

```diff
@@ -51,6 +51,49 @@ <h2>{{ site.blog_description }}</h2>
   </div>
   {% endif %}
 
+  {% assign featured_posts = site.posts | where: "featured", "true" %}
+  {% if featured_posts.size > 0 %}
+    <br>
+    <div class="container featured-posts">
+      {% assign is_even = featured_posts.size | modulo: 2 %}
+      <div class="row row-cols-{% if featured_posts.size <= 2 or is_even == 0 %}2{% else %}3{% endif %}">
+      {% for post in featured_posts %}
+        <div class="card-item col">
+          <a href="{{ post.url | relative_url }}">
+            <div class="card hoverable">
+              <div class="row g-0">
+                <div class="col-md-12">
+                  <div class="card-body">
+                    <div class="float-right">
+                      <i class="fa-solid fa-thumbtack fa-xs"></i>
+                    </div>
+                    <h3 class="card-title text-lowercase">{{ post.title }}</h3>
+                    <p class="card-text">{{ post.description }}</p>
+
+                    {% if post.external_source == blank %}
+                        {% assign read_time = post.content | number_of_words | divided_by: 180 | plus: 1 %}
+                      {% else %}
+                        {% assign read_time = post.feed_content | strip_html | number_of_words | divided_by: 180 | plus: 1 %}
+                      {% endif %}
+                      {% assign year = post.date | date: "%Y" %}
```

Comment:
nit: the indent is a bit off
```suggestion
                    {% if post.external_source == blank %}
                      {% assign read_time = post.content | number_of_words | divided_by: 180 | plus: 1 %}
                    {% else %}
                      {% assign read_time = post.feed_content | strip_html | number_of_words | divided_by: 180 | plus: 1 %}
                    {% endif %}
                    {% assign year = post.date | date: "%Y" %}
```